Fix bug file downloading

duylam opened the pull request:
Fix https://github.com/ovity/octotree/issues/339

Tested:
- Chrome on Mac OS
- Firefox on Mac OS

buunguyen commented:
Can you replace the download icon too? There are many icon fonts included, so there should be one that better communicates the intent of "open file in new tab".

duylam commented:
@buunguyen Addressed the comment, the new icon is https://octicons.github.com/icon/browser/, below is the screenshot

<img width="203" alt="screen shot 2018-10-12 at 22 31 36" src="https://user-images.githubusercontent.com/432049/46878986-05df1d80-ce6f-11e8-9ac6-717fdb7a1aaf.png">

duylam commented:
@buunguyen Addressed comments
